### ISSUE TITLE
Using your PT models with TTS by Coqui AI

### ISSUE THREAD

**Issue by fatualux** [simsax/italian-tts#1]
**Good morning** and **congrats** for your work.
I have been trying to use your models with Coqui TTS (_https://github.com/coqui-ai/TTS)_ but I couldn't get to find a way.

I tried this:

`(tts) [fz@fzpc ~]$ tts --text "Un testo a caso, qualunque esempio" --model_path "/home/fz/.local/share/tts/fastpitch/vits.pth" --out_path 1.wav
`
but with no luck.

the output to the command is:

`Traceback (most recent call last):
  File "/home/fz/.virtualenv/tts/bin/tts", line 8, in <module>
    sys.exit(main())
  File "/home/fz/.virtualenv/tts/TTS/TTS/bin/synthesize.py", line 371, in main
    synthesizer = Synthesizer(
  File "/home/fz/.virtualenv/tts/TTS/TTS/utils/synthesizer.py", line 86, in __init__
    self._load_tts(tts_checkpoint, tts_config_path, use_cuda)
  File "/home/fz/.virtualenv/tts/TTS/TTS/utils/synthesizer.py", line 144, in _load_tts
    self.tts_config = load_config(tts_config_path)
  File "/home/fz/.virtualenv/tts/TTS/TTS/config/__init__.py", line 79, in load_config
    ext = os.path.splitext(config_path)[1]
  File "/usr/lib/python3.9/posixpath.py", line 118, in splitext
    p = os.fspath(p)
TypeError: expected str, bytes or os.PathLike object, not NoneType
`

but I have not understood if it refers to the model itself.
In short words, I did not get to how to use your model, vits.pth.

I have TTS installed in a virtualenv,  on Arch Linux, latest kernel, python 3.11.

I am a noob, so please be patient :)

Thanks in advance for your support

**Comment by fatualux** [comment]
I think a .json config is missing

**Comment by simsax** [comment]
Hello! I've never tried using Coqui TTS with my own models, so unfortunately, I can't provide much assistance with that. However, I believe that it may not be possible to simply plug and play my checkpoints, since Coqui likely expects them to be in a different state. The checkpoints are intended to be used as explained in the readme. I hope this helps :)

**Comment by fatualux** [comment]
Thank you so much :)

..:FZ:..

- Sent from my Android device -

Il mar 9 mag 2023, 22:39 Simone Sasso ***@***.***> ha scritto:
